### ISSUE TITLE
RP breaks OpenComputers robots

### ISSUE THREAD

**Issue by Xandaros** [TheRandomLabs/RandomPatches#78]
With RP installed, [OpenComputers](https://www.curseforge.com/minecraft/mc-mods/opencomputers) robots are no longer able to mine blocks.

Unfortunately, I do not have the full error message any more, but it was a NoSuchMethodError and pointed at [`com.therandomlabs.randompatches.hook.PlayerInteractionManagerHook#sendBlockChangePacket`](https://github.com/TheRandomLabs/RandomPatches/blob/1.12/src/main/java/com/therandomlabs/randompatches/hook/PlayerInteractionManagerHook.java#L10)

Disabling the ghost block fix and doing `/rpreload` did not help, but restarting the server did.

This is in [Enigmatica 2 Expert Skyblock](https://www.curseforge.com/minecraft/modpacks/enigmatica2expertskyblock), in case some other mods are involved.

**Comment by TheRandomLabs** [comment]
Unfortunately, I can't seem to reproduce this issue on the latest version of Enigmatica 2: Expert Skyblock on either the client or the server with SpongeForge installed. Could I have a log the next time this happens?

**Comment by Xandaros** [comment]
Hmm, I just tried it in single player, since I don't want to restart the server right now, but it doesn't seem to show the error in the log. All I can offer is a screenshot: http://tcp.st/t01zy.png

Here's what I did:
Made a creative computer case, placed it and fitted it with all sorts of thing. I used this to install OpenOS on a hard drive.
Then put another creative case in an assembler and started assembling a robot. It has all the recommended parts and I also included an inventory upgrade.
I then assembled the robot, gave it a pickaxe and put a block in front of it.
I booted up the robot, executed `lua` to open a Lua REPL and tried to do `robot.swing()`.
Normally, this should destroy the block in front, but it doesn't. Instead you get a cryptic error message in-game.

On the server, I also got the aforementioned `NoSuchMethodError` on the console, though it might not have been every time.

You mentioned SpongeForge. I honestly have no idea what that is - I don't think I am using that. I simply downloaded the zip file for the modpack and gave it to MultiMC to work its magic.

If you need me to try it on the server to get the actual error message, I can do that tomorrow.

**Comment by TheRandomLabs** [comment]
Can you confirm that you are using Enigmatica 2: Expert Skyblock 1.32? Just making sure.

**Comment by TheRandomLabs** [comment]
This is in a singleplayer world.
![image](https://user-images.githubusercontent.com/16755704/72674026-1a748180-3ac6-11ea-8b6c-e7e19e26407a.png)

**Comment by Xandaros** [comment]
Ah! Sorry about that. We are on 1.31b. And the changelog for 1.32 says `Fixed instability caused by outdated RandomPatches version`...

Well, that probably settles that, then. If this is it, I'm sorry for wasting your time. I did check the RP version and thought it was up to date, but I guess I only compared major and minor.
I'll close this and try again after updating at a later date. Should the problem persist, I'll reopen.
